src/cli/commands/start/start.ts

```diff
@@ -298,4 +306,20 @@ export async function start(options: SWACLIConfig) {
     .catch((err) => {
       logger.error(err.message, true);
     });
+
+  const end = new Date().getTime();
+
+  collectTelemetryEvent(
+    TELEMETRY_START_EVENT,
+    {
+      macAddressHash: (await getMachineId("sha256", TELEMETRY_MAC_ADDRESS_HASH_LENGTH)).toString(),
```

Comment:
Also see if we can get the MAC Id once for a session and use for all the telemetry events